lib/command/update/general_lastup_updater.rb

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2013 whiteleaf. All rights reserved.
+#
+
+require_relative "../../narou/api"
+
+module Command
+  class Update < CommandBase
+    class GeneralLastupUpdater
+      def initialize(options)
+        @options = options
+        @database = Database.instance
+        @narou_novels = Hash.new { |h, k| h[k] = [] }
+        @other_novels = []
+        prepare
+      end
+
+      def save
+        @database.save_database
+      end
+
+      def prepare
+        @database.each.with_index do |(id, data), i|
+          next if Narou.novel_frozen?(id)
+          setting =  Downloader.get_sitesetting_by_target(id)
```

Comment:
__[rubocop]__
* Style/SpaceAroundOperators: Operator `=` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/ExtraSpacing: Unnecessary spacing detected.

<!-- Comment by [SideCI](https://sideci.com) -->